main.go

```diff
@@ -243,6 +245,8 @@ func goBuild(outputPath string) error {
 		return err
 	}
 
+	log.Printf("Building wasm output")
```

Comment:
Remove this (We might need such logs later but I am not 100% sure for now)

Reply:
fixed